test/integration/container/multimaster_test.go

```diff
@@ -367,6 +557,108 @@ func TestMultimasterSetup(t *testing.T) {
 	}
 }
 
+func TestMultimasterSetupUbuntu(t *testing.T) {
```

Comment:
This looks very similar to `TestMultimasterSetup` - could it be more DRY?

Reply:
Definitely :+1: